src/content/docs/plugins/submission.md

```diff
@@ -36,16 +38,12 @@ The submission process involves forking the Vencord repository, adding your plug
    ```bash
    mkdir myFirstPlugin
    ```
-3. Add your plugin files (`index.ts`, `README.md`, and optionally `native.ts`):
-    - **`index.ts`**: This is the main entry point for your plugin's code.
-    - **`README.md`**: Document your plugin, explain its functionality, and provide installation instructions and usage examples.
-    - **`native.ts`** (optional): If your plugin uses Node.js APIs, include this file.
-  
-   Make sure the `README.md` includes:
-   - A short description of your plugin.
-   - Instructions for installing and using the plugin.
+3. Ensure your `README.md` file is added and includes:
+   - A brief description of your plugin.
    - Screenshots, GIFs, or videos if applicable to showcase your plugin.
 
+> Since this is for a built-in plugin, no installation instructions are required.
```

Comment:
please dont use chatgpt for docs 😭

Reply:
I used it for fix it fast, as i dont had time rn, if want something better i can do it but first i should have more time to make it correct and dont use AI for corrections